Week3/homework/js-exercises/giveCompliment.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+function giveCompliment(name) {
+    const compliments = ["amazing", "great", "awesome", "beautiful", "clever", "strong", "perfect", "kind", "courageous", "fair-minded"];
+    const randomComp = compliments[Math.floor(Math.random() * compliments.length)];
+    return `You are ${randomComp} , ${name}`;
```

Comment:
👏 but the last element `fair-minded` can be written as `fair minded`, right? It would be weird sentence otherwise: `You are fair-minded, Ceylan` 😝

Reply:
Thanks @yash-kapila! Yes you are right, it is because of copy&paste 😅